Extensions/TextObject/textruntimeobject-pixi-renderer.ts

```diff
@@ -69,17 +69,23 @@ namespace gdjs {
         this._object._outlineColor[1],
         this._object._outlineColor[2]
       );
-      style.strokeThickness = this._object._outlineThickness;
+      style.strokeThickness = this._object._isOutlineEnabled
+        ? this._object._outlineThickness
+        : 0;
       style.dropShadow = this._object._shadow;
       style.dropShadowColor = gdjs.rgbToHexNumber(
         this._object._shadowColor[0],
         this._object._shadowColor[1],
         this._object._shadowColor[2]
       );
+      style.dropShadowAlpha = this._object._shadowOpacity / 255;
       style.dropShadowBlur = this._object._shadowBlur;
-      style.dropShadowAngle = this._object._shadowAngle;
+      style.dropShadowAngle = gdjs.toRad(this._object._shadowAngle);
```

Comment:
Pixi has the right referential for this? No need to add 90° or something like that?

Reply:
The shadow was where I expected it to be when I tested.